sdk/management/README.md

```diff
@@ -0,0 +1,221 @@
+- Web app and Function app
+- Key Vault
+- Cosmos
+
+## Examples
+
+### Fluent interface
+
+You can create a virtual machine instance, together with required virtual network and ip address.
+
+```java
+VirtualMachine linuxVM = azure.virtualMachines().define("myLinuxVM")
+	.withRegion(Region.US_EAST)
+	.withNewResourceGroup(rgName)
+	.withNewPrimaryNetwork("10.0.0.0/28")
+	.withPrimaryPrivateIPAddressDynamic()
+	.withNewPrimaryPublicIPAddress("mylinuxvm")
+	.withPopularLinuxImage(KnownLinuxVirtualMachineImage.UBUNTU_SERVER_16_04_LTS)
+	.withRootUsername("tirekicker")
+	.withSsh(sshKey)
+	.withSize(VirtualMachineSizeTypes.STANDARD_D3_V2)
+	.create();
+```
+
+Update.
+
+```java
+linuxVM.update()
+	.withNewDataDisk(20, lun, CachingTypes.READ_WRITE)
+	.apply();
+```
+
+### Asynchronous operations
+
+You can create storage account, then blob container, in reactive programming. 
+
+```java
+azure.storageAccounts().define("mystorageaccount")
+    .withRegion(Region.US_EAST)
+    .withNewResourceGroup(rgName)
+    .withSku(StorageAccountSkuType.STANDARD_LRS)
+    .withGeneralPurposeAccountKindV2()
+    .withOnlyHttpsTraffic()
+    .createAsync()
+    .filter(indexable -> indexable instanceof StorageAccount)
+    .last()
+    .flatMapMany(indexable -> azure.storageBlobContainers()
+        .defineContainer("container")
+        .withExistingBlobService(rgName, ((StorageAccount) indexable).name())
+        .withPublicAccess(PublicAccess.BLOB)
+        .createAsync()
+    ).blockLast();
+```
+
+### Configurable client
+
+You can customize various aspects of the client.
+
+```java
+Azure azure = Azure
+    .configure()
+    .withHttpClient(customizedHttpClient)
+    .withPolicy(additionalPolicy)
+    .withConfiguration(customizedConfiguration)
+    ...
+```
+
+### Include single package
+
+Instead of include the complete Azure Management Library, you can choose to include a single service package.
+
+For example, here is sample maven dependency for Compute package.
+
+[//]: # ({x-version-update-start;com.azure:azure-mgmt-compute;current})
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-mgmt-compute</artifactId>
+  <version>2.0.0</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+Sample code to create the authenticated client.
+```java
+ComputeManager client = ComputeManager.authenticate(credential, profile);
+client.virtualMachines().listByResourceGroup(rgName);
+```
+
+## Troubleshooting
+
+If you encounter any bugs, please file issues via [GitHub Issues](https://github.com/Azure/azure-sdk-for-java/issues/new/choose) 
+or checkout [StackOverflow for Azure Java SDK](http://stackoverflow.com/questions/tagged/azure-java-sdk).
+
+### HTTP client
+
+An `HttpClient` implementation must exist on the classpath.
+See [Include optional packages](#include-optional-packages).
+
+### Authentication
+
+By default, Azure Active Directory token authentication depends on correct configure of following environment variables.
```

Comment:
how about move this earlier, my point is that as a user, when i read or follow major steps in the readme, I can have a rough end to end idea of complete a mgmt task, from auth to config, to service features, then logging etc advanced features.

Reply:
Currently following other client README structure dictated by https://github.com/Azure/azure-sdk-for-java/blob/master/eng/.docsettings.yml#L16

When AUTH.md is ready, we can refer it in first line.

Reply:
I've moved this section to "Getting started" at top. Also added sample code.